les-nouvelles-de-l-archeologie.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Les nouvelles de l'archéologie (French)</title>
+    <title-short>NDA</title-short>
+    <id>http://www.zotero.org/styles/les-nouvelles-de-l-archeologie</id>
+    <link href="http://www.zotero.org/styles/les-nouvelles-de-l-archeologie" rel="self"/>
+    <link href="https://journals.openedition.org/nda/1554" rel="documentation"/>
+    <author>
+      <name>Sébastien Plutniak</name>
+      <email>sebastien.plutniak@posteo.net</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="history"/>
+    <issn>0242-7702</issn>
+    <eissn>2425-1941</eissn>
+    <summary>Style for the Les nouvelles de l'archéologie journal.</summary>
+    <updated>2019-10-06T10:57:00+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">
+        <single>(éd.)</single>
+        <multiple>(éds.)</multiple>
+      </term>
+      <term name="page" form="short">
+        <single/>
+        <multiple/>
+      </term>
+      <term name="no date">À paraître</term>
+    </terms>
+  </locale>
+  <!-- Macros definition -->
+  <macro name="editor">
+    <names variable="editor" delimiter=" ">
+      <name sort-separator=" " delimiter=", " name-as-sort-order="all" font-variant="small-caps" delimiter-precedes-last="never" initialize-with="."/>
+      <label form="short" prefix=" " suffix=" "/>
+      <substitute>
+        <names variable="editorial-director"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name and="symbol" form="short" delimiter=", " delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-long">
+    <names variable="author">
+      <name and="symbol" name-as-sort-order="all" font-variant="small-caps" sort-separator=" " delimiter=", " delimiter-precedes-last="never" initialize-with="."/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <names variable="editor">
+          <name and="symbol" name-as-sort-order="all" font-variant="small-caps" sort-separator=" " delimiter=", " delimiter-precedes-last="never" initialize-with="."/>
+          <substitute>
+            <names variable="editorial-director" font-variant="small-caps"/>
+          </substitute>
+        </names>
```

Comment:
```suggestion
        <names variable="editor"/>
        <names variable="editorial-director"/>
```

should do the same

Reply:
actually it'll give you a label such as "eds." for editors, which I think you want.

Reply:
Sorry, I don't understand this point (and haven't change these lines in the new version)

Reply:
Why are you using this complicated construct with two nested substitutes rather than just
```
<substitute>
  <names variable="editor"/>
  <names variable="editorial-director"/>
</substitute>
```

which isn't just cleaner, it should also produce more accurate output. If you check the current generated samples, the edited volume (Hancke et al.) doesn't have an editor label for editors, which is almost certainly incorrect.

Reply:
Thanks, now I got it. I modified the file as you suggest.